### DISCUSSION TITLE
@types/sketch

### DISCUSSION THREAD

**Discussion by alicerocheman** [DefinitelyTyped/DefinitelyTyped#55857] [Request a new @types package]
I'm working on a sketch plugin, and I couln't find any typing for the sketch document and its content.
It would be great to have such a package
[Sketch API reference](https://developer.sketch.com/reference/api/#sketch-components)

**Comment by hozblok** [comment]
Hi @alicerocheman ,

Is there a source code for this package somewhere, or is it just the documentation?

If I understand correctly, what you want is here:
https://github.com/sketch-community/sketch-types
or here:
https://www.npmjs.com/package/sketch-types
